src/desktop/main.js

```diff
@@ -216,7 +216,16 @@ function createWindow() {
         if (url.indexOf(targetURL) !== 0) {
             e.preventDefault();
 
-            const externalWhitelist = ['iota.org', 'docs.iota.works', 'trinity.iota.org', 'docs.bugsnag.com'];
+            const termsAndConditionsLinks = ['iota.org', 'trinity.iota.org', 'mailto:contact@iota.org'];
+            const privacyPolicyLinks = [
+                'corp.sogou.com',
+                'cryptocompare.com',
```

Comment:
Will this allow `www.cryptocompare.com`? That seems to be the URL in the privacy policy

Reply:
Yes, `www` subdomains are handled here https://github.com/iotaledger/trinity-wallet/blob/ff6dc0f60a50f2590c6019b47991f543a4bf443d/src/desktop/main.js#L232